bin/resource_management/resource-broken-links-csv.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import pandas as pd
+import urllib
+import sys
+from sys import stderr, stdout, argv
+from multiprocessing import Process
```

Comment:
these aren't CPU-bound tasks, an async approach instead of multiprocessing would be a lot better